loadstring(game:HttpGet("https://raw.githubusercontent.com/7yd7/Hub/refs/heads/Branch/Game"))()

--[[
	{
		title = "Warning",
		description = "We are not responsible if you are banned in the games that was used in the script"
	},

	{
		title = "Legal Notice",
		description = "Use of this script is at your own risk. The developer is not liable for any consequences arising from its usage"
	},
	{
		title = "Open source",
		description = "This is a completely open source script: ",
		link = "https://github.com/7yd7/Hub"
	},
	{
		title = "تحذير هام",
		description = "نحن غير مسؤولين عن أي حظر قد يحدث نتيجة استخدام هذا السكربت في أي ماب"
	},
	{
		title = "إخلاء مسؤولية",
		description = "استخدام هذا السكربت يكون على مسؤوليتك الشخصية. المطور غير مسؤول عن أي عواقب"
	},
	{
		title = "مفتوح المصدر",
		description = "هذا سكربت مفتوح المصدر بالكامل: ",
		link = "https://github.com/7yd7/Hub"
	}
]]
